Application/EdFi.Ods.AdminApi.AdminConsole/Infrastructure/DataAccess/Artifacts/Security/MsSql/20241217230931_AdminConsoleClaimset.cs

```diff
@@ -18,7 +23,7 @@ protected override void Up(MigrationBuilder migrationBuilder)
         /// <inheritdoc />
         protected override void Down(MigrationBuilder migrationBuilder)
         {
-
+            // No data down for this migration
```

Comment:
Can you create a bug ticket to implement this method?

Reply:
Do we really need a downgrade process? 

Why are we using Entity Framework at all? This is not our standard. Our standard is to use [migration scripts](https://github.com/Ed-Fi-Alliance-OSS/AdminAPI-2.x/tree/main/Application/EdFi.Ods.AdminApi/Artifacts) installed via our Ed-Fi DB Deploy tool.

Reply:
I was trying to remember why we leaned into EF. I think one reason was the fact that we have a flag in our appsettings file to enable and disable Admin Console Api endpoints. So we wanted to make this upgrade and downgrade more transparent, by running them at runtime.

Reply:
That makes sense... although I don't like this executing at runtime. I will think about this. I'm worried about our release time line, but I also don't like introducing a second way of handling database deployments.

Reply:
I created a ticket for later to think about moving away from EF.